sourcegraph/about
# Discord replaces Twitch in Footer

This closes #5566, and /community's [#18](https://github.com/sourcegraph/community/issues/18)

### Changelog
- Places Discord icon button where the Twitch was in the Footer
- Changes the `Community` Footer item name to `Discord`

### Open question
- @jdorfman , do we want community's slug to change to => `/discord` too? Or just the footer link to the community page?

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure the Discord icon renders & links to the correct page

#### Staging link:
https://deploy-preview-5570--sourcegraph.netlify.app/

<img width="896" alt="image" src="https://user-images.githubusercontent.com/59381432/178080245-63755094-d882-4eda-89e5-6006afa8e430.png">
